src/Datagrid/Pager.php

```diff
@@ -23,6 +24,11 @@
  */
 class Pager extends BasePager
 {
+    /**
+     * Use separator in CONCAT() function for correct determinate similar records.
+     */
+    public const CONCAT_SEPARATOR = '|';
```

Comment:
Why is this public?

Reply:
This was requested by @OskarStark https://github.com/sonata-project/SonataDoctrineORMAdminBundle/pull/996#discussion_r388132758